e2e/test/iothub/command/CommandE2ETests.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if !NET451
```

Comment:
why !net451? commands are supposed to be supported on net451

Reply:
DigitalTwinClient is not supported on 451...

![image](https://user-images.githubusercontent.com/73593426/119198413-48216000-ba57-11eb-9207-7657dd3dbcb1.png)

Reply:
we could use service client for net451 and digital twin for the others, or go back to service client for all (as you did have previously); your call. I thought using digital twin client would simplify the tests and make them more pnp-fied, but we do still want to test against all supported frameworks.